bigflow/workflow.py

```diff
@@ -135,11 +146,11 @@ def __init__(
             self.retry_pause_sec = retry_pause_sec
 
     @abc.abstractmethod
-    def execute(self, context: JobContext):
+    def execute(self, context: JobContext) -> Optional[Any]:
         raise NotImplementedError
 
     @public(deprecate_reason="Method `Job.run` is deprecated, use `Job.execute()` instead")
-    def run(self, runtime):
+    def run(self, runtime: Union[str, dt.datetime, dt.date, None]) -> Any:
```

Comment:
`datetime` is a subclass of `date`

Reply:
Good to know :) I think that redundant, explicit datetime is ok, to make things more clear.